.github/workflows/build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build and Test
+
+on:
+ push:
+   branches: [master]
+   tags:
+ pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+          python-version: ['2.7', '3.5', 'pypy2.7']
```

Comment:
this yaml is indented wrong it looks like

Reply:
deindented this.